pkg/manifests/types.go

```diff
@@ -253,6 +253,9 @@ type ThanosQuerierConfig struct {
 type NodeExporterConfig struct {
 	// Defines which collectors are enabled and their additional configuration parameters.
 	Collectors NodeExporterCollectorConfig `json:"collectors,omitempty"`
+	// Number of processes Node Exporter runs in parallel.
+	// After 1.5.0 Node Exporter default to 1 process, the default value here is 0, retaining Node Exporter's old behavior before 1.5.0.
```

Comment:
I don't think that we need to go into the details of node_exporter's version. Maybe we should add why you may want to change this value (e.g. limit the concurrency and avoid data races and I/O issues on machines with high number of CPUs) and what are the consequences (longer times to collect the metrics)?
cc @bburt-rh 

```suggestion
	// By default, Node Exporter is configured to use all node's CPUs.
```

Reply:
@simonpasquier We've added some additional info around concurrency and potential I/O issues.